### PR TITLE
Fix charts totals

### PR DESCRIPTION
**Pull request by twheys**
Some cases of combinations of dimensions with rollup were not covered by the last change to the way rolled up dimensions are handled.

### CODE REVIEW

File: fireant/slicer/widgets/highcharts.py [hunk 1]
```diff
@@ -148,6 +148,23 @@ def transform(self, data_frame, slicer, dimensions, references):
             "legend": {"useHTML": True},
         }
```
[the remark this answers is not in the document]
Comment (twheys) [reply fireant/slicer/widgets/highcharts.py]: There's a big untested method right now in the module database.py which is where the bug originated from. I've added another test case to cover the highcharts module but there are still a few untested corner cases.

[the remark this answers is not in the document]
Comment (mikeengland) [reply fireant/slicer/widgets/highcharts.py]: Cool, looks good 👍
